features/schema.py

```diff
@@ -185,6 +194,7 @@ class FeatureDetails(ObjectType):
     """Detailed information a Feature might have."""
 
     harbor = graphene.Field(HarborDetails, description=_("Details of a harbor"))
+    priceList = graphene.List("features.schema.PriceTag", required=True)
```

Comment:
convention is to name fields along with the regular PEP standards, i.e. `price_list`. graphene will generate camelcase itself for the schema.

Reply:
fixed